src/Migrations/20200203143917-create-minig-rewards-table.js

```diff
@@ -0,0 +1,31 @@
+var sequelize = require("sequelize");
+
+module.exports = {
+    up: function (queryInterface) {
+        return queryInterface.createTable('MiningRewards', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: sequelize.INTEGER
+            },
+            minerRewards: {
+                type: sequelize.ARRAY(sequelize.JSON)
+            },
+            rewardAmount: {
+                type: sequelize.INTEGER
```

Comment:
This should be a big int. Check with @MakMuftic are there any changes regarding defining big integers.